### DISCUSSION TITLE
81E...不能用？

### DISCUSSION THREAD

**Discussion by qiu2020** [Navid200/xDrip#51] [Dexcom]
![image](https://user-images.githubusercontent.com/49631431/126890135-e248fffb-90bf-4f40-9c2c-014749ac1d20.png)

**Comment by Navid200** [comment]
This is a very old transmitter, based on the serial number.
Is the battery a replaced battery or a rechargeable battery?  Or, is this the original battery?  That would be astonishing.

What version of xDrip is this?  You can see on the classic status page.  

Are the settings all correct?
https://github.com/Navid200/xDrip/wiki/G6-Recommended-Settings

**Comment by Navid200** [comment]
You should also disable the preemptive restart setting for this transmitter.  
https://github.com/Navid200/xDrip/wiki/Preemptive-Restart

Would you please show the logs as well?
